content/lessons/chapter-6/in-out-4.tsx

```diff
@@ -200,6 +200,9 @@ class Input:
           <Title>{t('chapter_six.in_out_four.heading_three')}</Title>
         </div>
         <Table headings={tableHeading} rows={inputRows} />
+        <Text className="mt-4 text-lg md:text-3xl">
```

Comment:
```suggestion
        <Text className="mt-4 font-nunito text-xl text-white">
```

Did you mean for this bit ("And remember...") to be big? It looks a little out of place next to the rest of the challenge copy:

![Screenshot from 2024-05-29 23-13-21](https://github.com/saving-satoshi/saving-satoshi/assets/1823216/8d1f8a3d-9ab5-4eec-a1c4-a9ebad9bf353)

Reply:
Haha yes I intentionally blew that up to make it "loud" -- is that annoying? I can put it back to the default font size

Reply:
Got it :D What about making it the same size/weight as the "Outpoint" and "Input", so `font-nunito text-2xl font-bold text-white`?

Reply:
done, here and in the next page as well